choreographer/src/main/java/eu/arrowhead/core/choreographer/service/ChoreographerService.java

```diff
@@ -176,15 +188,16 @@ private void sendNotification(final ChoreographerSession session, final String m
     	logger.debug("sendNotification started..."); 
     	
     	try {
-    		if (!Utilities.isEmpty(session.getNotifyUri())) {
-    			final ChoreographerNotificationDTO payload = new ChoreographerNotificationDTO(session.getId(),
-    																						  session.getPlan().getId(),
-    																						  session.getPlan().getName(),
+    		final ChoreographerSession _session = sessionDBService.getSessionById(session.getId()); //refresh
```

Comment:
maybe we should use only sessionId as input, then we can rename _session to session